popup.html

```diff
@@ -1,17 +1,18 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>WakaTime</title>
-
-    <link href="public/css/app.css" rel="stylesheet">
-</head>
-<body>
-
-<div id="wakatime"></div>
-
-<script src="public/js/app.js"></script>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WakaTime</title>
+
+    <link href="public/css/app.css" rel="stylesheet">
+    <script src="public/js/browser-polyfill.min.js"></script>

```

Comment:
Wouldn't it be better to load the polyfill as part of the header instead of as part of the body? From what I understand it's usually best practice to load script files in the header and just use the DomContentLoaded event to actually start working with it.